.github/workflows/test_merge_requests.yml

```diff
@@ -34,7 +35,7 @@ jobs:
       # Must not be affected by file metadata changes and have a consistent sort order:
       # https://docs.github.com/en/actions/learn-github-actions/expressions#hashfiles
       # Keying by the relevant build context is more re-usable than a commit SHA.
-      - name: 'Derive Docker image cache key from content'
+      - name: Derive Docker image cache key from content
```

Comment:
> This was discussed between me and @casperklein [earlier](https://github.com/docker-mailserver/docker-mailserver/pull/2742#issuecomment-1226724889), you did not provide your input regarding it.

Sorry, must have overlooked that. I would like to stick to "only use quotes when needed" for two reasons:

1. the linters for YAML I use and that I know of use this as a default
2. in our current code base, this is actually the way it is done for most strings -- yes, many strings that are to be read by humans are quotes, but for many other strings, quotes are missing, which _should_ in theory also have quotes if we stick to "use quotes everywhere". I have prepared a PR which solves the quotes-style-problem (upcoming).


> 
> It would be good to come to agreement here, personally I prefer quote wrapping the value (_it's syntax highlighted as a string in my editor this way, and usually a habit of mine due to some yaml gotchas with like with our port numbers in `docker-compose.yml`_). As an added bonus it tends to help each step stand out a bit visually when glancing over in the editor :) I don't mind the other way too, but then I would like to see a PR that handles this as well :)

Reply:
> which should in theory also have quotes if we stick to "use quotes everywhere".

For the current [`test_merge_requests.yml`](https://github.com/docker-mailserver/docker-mailserver/blob/e6f481b59f7f67a7f605f9d97fa6a5ee1de68840/.github/workflows/test_merge_requests.yml) I've only used single quote wrapping on step `name`. This is consistent in that usage for steps, rather than "quotes everywhere".

I found it helpful for syntax highlighting steps visually apart better:

![Screenshot_20220831_184718](https://user-images.githubusercontent.com/5098581/187611841-02e5f1f5-20e3-49c2-be13-b390f4bb7cc3.png)

but I guess that depends on editor / theme support from the syntax-highlighter:

![Screenshot_20220831_184830](https://user-images.githubusercontent.com/5098581/187612095-04df8b29-a85f-48c1-b451-e39adec6d8a6.png)

It's my preference but I'm fine with dropping the quotes if you want to maintain consistency for YAML fields in general.

Reply:
Ah, I see. I am using VS Code :) I am fine with having the quotes around the `name:` fields if this provides better readability. I will add them to the files touched by this PR. A follow-up PR may provide consistency for _all_ files.

Reply:
See 04bac22b1c212fa41c2d1f36bf166a004073917c